DocAI Incubator Tools/best-practices/Removing_Empty_Bounding_Boxes.ipynb

```diff
@@ -0,0 +1,420 @@
+{
```

Comment:
<p>Similar comment to before, a lot of this code is very difficult to read, could be refactored for easier understanding</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/document-ai-samples/pull/640/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>